script/standalone_ipv6

```diff
@@ -0,0 +1,335 @@
+#! /bin/sh
+#
+#  Copyright (c) 2017, The OpenThread Authors.
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#  1. Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#  2. Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#  3. Neither the name of the copyright holder nor the
+#     names of its contributors may be used to endorse or promote products
+#     derived from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+#----------------------------------------
+# Purpose:
+#  To understand the purpose of this script see: print_big_ugly_warning() below.
+#----------------------------------------
+#   
+
+# remember the name of this script
+SCRIPT_NAME=$0
+
+CWD=`pwd`
+DATE=`date`
+
+_checkfile=./src/agent/main.cpp
+if [ ! -f $_checkfile ]
+then
+    echo "ERROR: Cannot find checkfile: ${_checkfile}"
+    echo "This script should be executed from ROOT of the border router directory"
+    echo "For example:"
+    echo ""
+    echo "cd \${BORDERROUTER}"
+    echo "sh ./script/bootstrap"
+    echo ""
+    exit 1
+fi
+
+# source our common stuff
+. "$(dirname "$0")"/_initrc
+
+OPENTHREAD_ONMESH_PREFIX=fd11:22
+ETH0_IPV6_BASE_PREFIX=fd11:33
+
+debug_echo() {
+    #echo "${@}"
```

Comment:
use a environment to enable or disable debug message?

Reply:
done